core/src/themes/ionic/ionic.theme.default.scss

```diff
@@ -10,85 +10,211 @@
+      ),
+    ),
   warning: (
-    base: $warning,
-    contrast: #000,
-    shade: color.get-color-shade($warning),
-    tint: color.get-color-tint($warning),
+    bold: (
+      base: globals.$ion-bg-warning-base-default,
+      base-rgb: globals.$ion-bg-warning-base-default-rgb,
+      contrast: globals.$ion-text-default,
+      contrast-rgb: globals.$ion-text-default-rgb,
+      shade: globals.$ion-bg-warning-base-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-primitives-yellow-300,
+      foreground: globals.$ion-text-warning,
+    ),
+    subtle: (
+      base: globals.$ion-bg-warning-subtle-default,
+      base-rgb: globals.$ion-bg-warning-subtle-default-rgb,
+      contrast: globals.$ion-text-warning,
+      contrast-rgb: globals.$ion-text-warning-rgb,
+      shade: globals.$ion-bg-warning-subtle-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-primitives-yellow-100,
+      foreground: globals.$ion-text-warning,
+    ),
   ),
   danger: (
-    base: $danger,
-    contrast: #fff,
-    shade: color.get-color-shade($danger),
-    tint: color.get-color-tint($danger),
-  ),
-  light: (
-    base: $light,
-    contrast: #000,
-    shade: color.get-color-shade($light),
-    tint: color.get-color-tint($light),
-  ),
-  medium: (
-    base: $medium,
-    contrast: $medium,
-    shade: color.get-color-shade($medium),
-    tint: color.get-color-tint($medium),
-  ),
-  neutral: (
-    base: $neutral,
-    contrast: #fff,
-    shade: color.get-color-shade($neutral),
-    tint: color.get-color-tint($neutral),
-  ),
-  dark: (
-    base: $dark,
-    contrast: $dark,
-    shade: color.get-color-shade($dark),
-    tint: color.get-color-tint($dark),
+    bold: (
+      base: globals.$ion-bg-danger-base-default,
+      base-rgb: globals.$ion-bg-danger-base-default-rgb,
+      contrast: globals.$ion-text-inverse,
+      contrast-rgb: globals.$ion-text-inverse-rgb,
+      shade: globals.$ion-bg-danger-base-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-semantics-danger-700,
+      foreground: globals.$ion-text-danger,
+    ),
+    subtle: (
+      base: globals.$ion-bg-danger-subtle-default,
+      base-rgb: globals.$ion-bg-danger-subtle-default-rgb,
+      contrast: globals.$ion-text-danger,
+      contrast-rgb: globals.$ion-text-danger-rgb,
+      shade: globals.$ion-bg-danger-subtle-press,
+      // TODO this is not a verified value
+      tint: globals.$ion-semantics-danger-200,
+      foreground: globals.$ion-text-danger,
+    ),
   ),
+  // neutral/base equivalent
+  light:
+    (
+      bold: (
+        base: globals.$ion-bg-neutral-base-default,
+        base-rgb: globals.$ion-bg-neutral-base-default-rgb,
+        contrast: globals.$ion-text-default,
+        contrast-rgb: globals.$ion-text-default-rgb,
+        // TODO this fails AA testing
+        // shade: globals.$ion-bg-neutral-base-press,
+        shade: globals.$ion-primitives-neutral-600,
```

Comment:
The 600 token is one step lighter but it fails AA with default text

Reply:
Maybe use `--ion-color-dark-subtle-contrast` or `--ion-bg-neutral-bold-press?`

Reply:
I sent the colors to design and they are going to take a look at them so I am sure they will get back to me on this one.